binstar_client/utils/__init__.py

```diff
@@ -247,8 +245,8 @@ def recursive_update(d, u):
             d[k] = u[k]
     return d
 
-DEFAULT_URL = 'https://api.anaconda.org'
```

Comment:
how about moving [1] to the top?
https://github.com/Anaconda-Server/anaconda-client/blob/api-on-premises/binstar_client/utils/__init__.py#L248-L250